src/style.css

```diff
@@ -1 +1,27 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__stars {
+  content: '';
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
Try to combine these only with `background` property 
<img width="304" alt="image" src="https://github.com/user-attachments/assets/901b37e4-7381-47df-bdc7-b5aa05bbac9f">